R/shinyui.R

```diff
@@ -69,7 +70,6 @@ jqueryDependency <- function() {
     return(htmlDependency(
       "jquery", version_jquery,
       src = c(
-        href = "shared",
         file = "www/shared"
```

Comment:
I believe we'd need a single dependency that is `all_files = TRUE` for the `www/shared` folder.

Now that we're also not using `href` to define the location, should we move jQuery, jQuery legacy, shiny.js, and all other html deps that do not need to be in the `shared` folder out of `www/shared`? (... since they are directly imported, no need to _double_ import)

Reply:
> I believe we'd need a single dependency that is all_files = TRUE for the www/shared folder.

I don't think that's good a idea: other packages also include libraries like jQuery, ionRangeSlider, etc., and by bundling them together, we'd open the possibility of loading two, possibly different versions, of the same library

> should we move jQuery, jQuery legacy, shiny.js, and all other html deps that do not need to be in the shared folder out of www/shared? 

Seems reasonable to me

Reply:
I don't think we should move them out of `shared` at this point. If we do that, it would break any apps that supply their own `index.html` with hard-coded paths, like the [`08_html`](https://github.com/rstudio/shiny/blob/ed547fdf406a0518d20c078ebfd547559c173b8d/inst/examples/08_html/www/index.html) example.

If we do make that change, we should do it in a separate PR and figure out a way to provide a good migration path for users.

Reply:
Oh, and I agree that we should should use `all_files=FALSE`. I don't think anyone would use the html dependencies in the top-level `www/shared/` dir for statically-rendered documents (that dir just contains jquery, shiny, and shiny-showcase files), but just in case, `all_files=FALSE` wouldn't hurt.